### ISSUE TITLE
It is difficult to distinguish the links within the text

### ISSUE THREAD

**Issue by victorhck** [agama-project/agama-project.github.io#4]
It's difficult to know that a text is an inline link.

Example:

![image](https://github.com/user-attachments/assets/2ce340b2-61bf-40a7-bf00-55957ff0ee4f)

Without hover the mouse in the text you can't discover the links.

Greetings.

**Comment by imobachgs** [comment]
Yes, there are some problems in the styles, links should be underlined. We will have a look. Thanks!

**Comment by lslezak** [comment]
Fixed in #5, closing the issue.
